<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>条码扫描</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/images.qrc">
    <normaloff>:/logo/logo64.ico</normaloff>:/logo/logo64.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_scanning">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>541</width>
      <height>118</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>q123</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_scanning">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>39</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>条码扫描:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>841</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_previous_barcode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>39</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>刻印条码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_previous_barcode">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>541</width>
      <height>118</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>680</y>
      <width>451</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PLC的IP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_IP">
       <property name="text">
        <string>192.168.0.255</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_change_ip">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1.2, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(200, 194, 15);</string>
       </property>
       <property name="text">
        <string>设置为当前值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_plc_connect_test">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1.2, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(200, 194, 15);</string>
       </property>
       <property name="text">
        <string>测试连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_next_barcode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>39</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>待刻条码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_next_barcode">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>541</width>
      <height>118</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit_scanning</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>barcode_scanning()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_change_ip</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>change_ip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>969</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>967</x>
     <y>712</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_plc_connect_test</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connect_test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1072</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>1073</x>
     <y>726</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>barcode_scanning()</slot>
  <slot>change_ip()</slot>
  <slot>connect_test()</slot>
 </slots>
</ui>
